src/test/resources/mappings/1.2.5-intermediary.tiny

```diff
@@ -0,0 +1,8905 @@
+v1	intermediary	officialClient	officialServer
```

Comment:
Can you add a small file next to this explaining where it came from and its license?